modules/registrars/realtimeregister/additionalfields.php

```diff
@@ -105,4 +106,15 @@ function ($field) {
     }
 }
 
+/* Add custom fields here, example
```

Comment:
Is this an example for an override(translation), or for an extra field, which doesn't come from Realtime Register?

Reply:
extra field yes